test/cppsuite/test_harness/random_generator.h

```diff
@@ -58,6 +59,14 @@ class random_generator {
         return (random_string);
     }
 
+    /* Generate a random number between min and max. */
```

Comment:
Cann we call this `generate_integer` so its a bit more explicit? And update the comment to random integer.